### PR TITLE
Sign out and login UI

### PR DESCRIPTION
**Pull request by lkdfount**
- fixed the profile page (added to the index.js file so when the user routes to /Profile they go to the profile page)
- fixed the UI of the log in page
- added sign out button with correct functionality to the profile page

### CODE REVIEW

File: travvy/src/App.vue [hunk 1]
```diff
@@ -160,8 +160,8 @@ border-radius: 10px;
 }
 
 .info {
- background-color: #89C4A0;
-  color: white;
+ background-color: #C7EEA9;
+  color: black;
```
Comment (emilynairnn) [inline travvy/src/App.vue:164]: Thanks for fixing this! Hopefully the users will be able to read the text better now.

Comment (lucasungaretti) [reply travvy/src/App.vue:164]: agreed, thats good implementation from the feedback the TAs gave us

File: travvy/src/router/index.js [hunk 2]
```diff
@@ -7,6 +7,7 @@ import AttractionDetails from '../components/AttractionDetails.vue'
 import SignUp from '../components/SignUp.vue'
 import LogIn from '../components/LogIn.vue'
 import Onboarding from '../components/Onboarding.vue'
+import Profile from '../components/Profile.vue'
```
Comment (lucasungaretti) [inline travvy/src/router/index.js:10]: good catch

**Review by lucasungaretti — COMMENTED** [review]
LGTM :)

**Pull request merged**